packages/@aws-cdk/aws-ecs/lib/container-definition.ts

```diff
@@ -443,7 +443,7 @@ export class ContainerDefinition extends CoreConstruct {
    * Whether this container definition references a specific JSON field of a secret
    * stored in Secrets Manager.
    */
-  public readonly referencesSecretJsonField?: boolean;
+  public referencesSecretJsonField?: boolean;
```

Comment:
This property should only be modifiable from within the class. Can you replace this with a public property getter that references a private property?

Reply:
Sure thing, will push a commit.